packages/server/modules/workspaces/tests/e2e/Query.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ApolloServer } from 'apollo-server-express'
```

Comment:
The objection being that E2E testing would usually involve hitting the endpoints from the outside of the app, essentially running it in real/production circumstances. This might seem pretty close, but its still just creating an ApolloServer instance inside of the test process and not making any HTTP calls whatsoever.

E2E tests are also expected to run way longer and require custom infrastructure setup, which is also not the case for these - they can be ran the same way as other integration tests and the app doesn't even have to be built